### ISSUE TITLE
Show full option name in dropdowns

### ISSUE THREAD

**Issue by Eddcapone** [HeidiSQL/HeidiSQL#2006]
**Is your feature request related to a problem? Please describe.**
Options in dropdowns are not fully readable:

![image](https://github.com/user-attachments/assets/0e8c4f6c-7ea9-49e2-b0de-e50560e77f46)

**Describe the solution you'd like**
Either make the dropdown as wide as the longest option.

**Describe alternatives you've considered**
Show the full option name on hover.

**Additional context**
-

**Comment by ansgarbecker** [comment]
You can widen the dropdown menu by widening the column header:

![grafik](https://github.com/user-attachments/assets/34e8e89e-fce3-49d4-b4ea-d6cc827b7376)

**Comment by Eddcapone** [comment]
Thx, this is good enough for me :)

Maybe you could also make it show it on hover.

**Comment by ansgarbecker** [comment]
I just saw there is `TComboBox.AutoDropDownWidth`, which resizes the combobox to the widest item text. But that looked a bit strange, as it sticks out of the field then, probably a lot.
So I made an attempt to show a mouse-over hint, which can be found in the latest build now:

![grafik](https://github.com/user-attachments/assets/a0674824-f48a-4c07-8d44-e6fd44eacd40)

I'm sure this thing has new bugs. However, looks promising and useful to me.
